### PR TITLE
Feature/update icons

### LINKED ISSUE #71: Launcher icon
- [x] Launcher icon
- [x] Notification Icon for Android Status Bar 
- [ ] Theme for first Activity (See Riot)

### PR DESCRIPTION
**Pull request by BillCarsonFr**
Fixes #71 
Updated launcher icons and status bar

image

image

### CODE REVIEW

File: vector/src/main/java/im/vector/riotredesign/features/notifications/NotificationUtils.kt [hunk 1]
```diff
@@ -376,7 +376,7 @@ object NotificationUtils {
         val accentColor = ContextCompat.getColor(context, R.color.notification_accent_color)
         // Build the pending intent for when the notification is clicked
         val openRoomIntent = buildOpenRoomIntent(context, roomInfo.roomId)
-        val smallIcon = if (roomInfo.shouldBing) R.drawable.icon_notif_important else R.drawable.logo_transparent
+        val smallIcon = R.drawable.ic_status_bar
```
Comment (bmarty) [inline vector/src/main/java/im/vector/riotredesign/features/notifications/NotificationUtils.kt]: no more bing icon then?

Comment (BillCarsonFr) [reply vector/src/main/java/im/vector/riotredesign/features/notifications/NotificationUtils.kt]: nop :/, we could use a kind of level maybe?

**Comment by vigonotion** [comment]
I absolutely love the new status bar icon, but why did you change the app icon like that? With the long shadow, it looks like an app icon from android 5, the last icon without shadow looked far more modern. Also, the icon is rather pixelated on my device:

icon

**Comment by bmarty** [comment]
@vigonotion we will check why it's pixelated, thanks for the feedback

**Pull request merged**